ui/component/walletExport/view.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import { Form, FormField } from 'component/common/form';
+import Card from 'component/common/card';
+import Lbry from 'lbry';
+import { clipboard } from 'electron';
+import * as ICONS from 'constants/icons';
+import FileExporter from 'component/common/file-exporter';
+
+type Props = {
+  toast: (string, boolean) => void,
+};
+
+function WalletExport(props: Props) {
+  const { toast } = props;
+
+  const [password, setPassword] = React.useState();
+  const [data, setData] = React.useState();
+  const [fetching, setFetching] = React.useState();
+  const getData = async (password) => {
+    setFetching(true);
+    const data = await Lbry.sync_apply({ password });
+    setFetching(false);
+    setData(data);
+  };
+
+  return (
+    <Card
+      title={__('Export Wallet')}
+      subtitle={
+        data
+          ? __('Your Sync Data is prepared. You can now copy or save it.')
+          : __('Export wallet sync data. Choose a secure password you will need for import.')
+      }
+      actions={
+        <React.Fragment>
+          {!data && (
+            <Form onSubmit={() => alert()}>
+              <FormField
+                autoFocus
+                type="password"
+                name="password_set"
```

Comment:
- [ ] Site passwords get prompt for update

Every time I entered the password for encryption, Chrome tries to update the saved passed to the site.  Not sure if it's due to `type` or something else -- we do use `type="password"` elsewhere and never seen this behavior, so maybe something else.

<img width="610" alt="image" src="https://user-images.githubusercontent.com/64950861/179444813-dc92874d-e7ce-45df-b555-36fa0b9c25df.png">